src/main/kotlin/day03/Hero.kt

```
@@ -0,0 +1,44 @@
+
+class Hero(
+    // var : 안에서 접근 가능
+    // val : 읽기 전용
+    var name: String,
+    // 초기값
+    var hp: Int = 100,
+) {
+    var food = ""
+
+    /// 메소드
+    fun attack() {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)